serve/mlc_serve/engine/base.py

```diff
@@ -27,7 +27,6 @@ class MLCServeEngineConfig:
     max_num_batched_tokens: int = -1
     min_decode_steps: int = 32
     max_decode_steps: int = 48
-    prompt_allocate_ratio: float = 2.0
```

Comment:
Why remove this? I remember this is one of perf tuning knobs. So it is not supposed to be removed without a good reason.

Reply:
answer in this https://github.com/octoml/mlc-llm/pull/89#issuecomment-1828641261 comment.